packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -83,11 +84,13 @@ def info(self, dcos_api_session):
         return self._info
 
     def hostport(self, dcos_api_session):
+        """ returns container ip and port for calico and dcos network otherwise the ones from the host
+        """
         info = self.info(dcos_api_session)
         task = info['app']['tasks'][0]
         if 'networks' in self.app and \
                 self.app['networks'][0]['mode'] == 'container' and \
-                self.app['networks'][0]['name'] == 'dcos':
+                self.app['networks'][0]['name'] in ["calico", "dcos"]:
```

Comment:
This change is to make a minimal change to `test_networking.py` to influence other tests like `test_dcos_cni_l4lb`
https://github.com/dcos/dcos/pull/6582/files#diff-89643056102d7647724e7dc94be80843R793